### DISCUSSION TITLE
non-important typo on advisory for CVE-2023-28321

### DISCUSSION THREAD

**Discussion by samueloph** [curl/curl#12004] [General]
I'm not sure what's the best place to report this, so I'm doing it here.

The advisory says:
> ... removes the support for "partial" patches and ...

And I think it meant to say:

> ... removes the support for "partial" matches and ...

s/patches/matches

https://curl.se/docs/CVE-2023-28321.html

**Comment by bagder** [comment]
Thanks, fixed now in https://github.com/curl/curl-www/commit/fb6276ad2c07e67a9d198408f0ca0d96bdbb3c03
